Jenkinsfile

```
@@ -11,14 +24,14 @@ pipeline {
             matrix {
                 axes {
                     axis {
-                        name 'AGENT_TYPE'
-                        values 'linux', 'windows-2019', 'windows-2022'
+                        name 'IMAGE_TYPE'
+                        values 'linux', 'nanoserver-1809', 'nanoserver-ltsc2022', 'windowsservercore-ltsc2019', 'windowsservercore-ltsc2022'
```

To fix #451 there will be an additional PR containing this change:
```suggestion
                        values 'linux', 'nanoserver-1809', 'nanoserver-ltsc2019', 'nanoserver-ltsc2022', 'windowsservercore-ltsc2019', 'windowsservercore-ltsc2022'
```

Not sure if it will be needed but we could also add a `windowsservercore-1809` "proper" docker agent image with the following change:
```suggestion
                        values 'linux', 'nanoserver-1809', 'nanoserver-ltsc2022', 'windowsservercore-1809', 'windowsservercore-ltsc2019', 'windowsservercore-ltsc2022'
```

---

Previous build containing these 2 additional sets of images: https://ci.jenkins.io/job/Packaging/job/docker-agent/job/PR-459/55/

---

> we could also add a "proper" windowsservercore-1809 docker agent image

Rethinking about it, we should propose this new image in this PR IMO, so we can mention in the release note that if you want to keep a WSC image as close as the previous one you can use this new `windowsservercore-1809` image to avoid breaking change.

---

`windowsservercore-1809` image added.

---

`nanoserver-ltsc2019` addition in https://github.com/jenkinsci/docker-agent/pull/488